internal/services/users/user_resource_test.go

```diff
@@ -171,7 +171,7 @@ resource "azuread_user" "manager" {
 }
 
 resource "azuread_user" "test" {
-  user_principal_name = "acctestUser.%[1]d@${data.azuread_domains.test.domains.0.domain_name}"
+  user_principal_name = "acct'estUser.%[1]d@${data.azuread_domains.test.domains.0.domain_name}"
```

Comment:
```suggestion
  user_principal_name = "acctestUser'%[1]d@${data.azuread_domains.test.domains.0.domain_name}"
```

Reply:
Right, I see what you mean. Seems I overlooked those! Will change to match the above diff asap. Thanks.

Reply:
PS: I have "Allow edits by maintainers" ticked. Unsure why you're not allowed to push. If there's any other setting in my repo fork that I should toggle, please let me know!